contracts/oracles/MakerDAOOracleV2.sol

```diff
@@ -0,0 +1,126 @@
+pragma solidity 0.5.8;
+
+import "../interfaces/IOracle.sol";
+import "../external/IMedianizer.sol";
+import "../interfaces/IPolymathRegistry.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "../interfaces/IModule.sol";
+import "../interfaces/ISecurityToken.sol";
+import "../interfaces/ISecurityTokenRegistry.sol";
+
+contract MakerDAOOracleV2 is IOracle, Ownable {
+    IMedianizer public medianizer;
+    address public currencyAddress;
+    IPolymathRegistry polymathRegistry;
+    bytes32 public currencySymbol;
+
+    bool public manualOverride;
+    uint256 public manualPrice;
+
+    string constant STR = "SecurityTokenRegistry";
+
+    /*solium-disable-next-line security/no-block-members*/
+    event ChangeMedianizer(address _newMedianizer, address _oldMedianizer);
+    event SetManualPrice(uint256 _oldPrice, uint256 _newPrice);
+    event SetManualOverride(bool _override);
+
+    /**
+      * @notice Creates a new Maker based oracle
+      * @param _medianizer Address of Maker medianizer
+      * @param _currencyAddress Address of currency (0x0 for ETH)
+      * @param _currencySymbol Symbol of currency
+      * @param _polymathRegistry Address of the Polymath Registry
+      */
+    constructor(address _medianizer, address _currencyAddress, bytes32 _currencySymbol, address _polymathRegistry) public {
+        medianizer = IMedianizer(_medianizer);
+        currencyAddress = _currencyAddress;
```

Comment:
Why do we need this?

Reply:
To know what is the currency address for which we created a wrapper oracle